racingcar-java/src/main/java/Car.java

```diff
@@ -0,0 +1,17 @@
+public class Car {
+
+    private static final int CONDITION_MOVE = 4;
+
+    private int position = 0;
```

Comment:
```suggestion
    private static final int DEFAULT_POSITION = 0;
    
    private int position = DEFAULT_POSITION;
```
으로 매직넘버를 또 처리해줄 수 있겠네요 :)